enterprise-suite/templates/es-console.yaml

```diff
@@ -24,7 +24,8 @@ spec:
           configMap:
             name: es-console
 
-{{ if .Values.minikube }}
+{{ if or .Values.minikube .Values.exposeConsoleService }}
```

Comment:
I don't think you're reading the feedback on this one properly.  This fixed node port expose service must only be defined on minikube.  It must not be defined for anything else.